packages/client/src/test.cairo

```diff
@@ -35,7 +35,7 @@ struct UtreexoArgs {
 /// Receives arguments in a serialized format (Cairo serde).
 /// Panics in case of a validation error or chain state mismatch.
 /// Prints result to the stdout.
-fn test(mut arguments: Span<felt252>) {
+fn test(mut arguments: Span<felt252>, execute_script: bool) {
```

Comment:
I'd probably do that via feature, wdyt?

Reply:
The cost of single check per block is negligible, while features are a pain to use without something like cfg! macro.